core/src/services/cos/backend.rs

```diff
@@ -277,7 +277,11 @@ impl Accessor for CosBackend {
                 // The max multipart size of COS is 5 GiB.
                 //
                 // ref: <https://www.tencentcloud.com/document/product/436/14112>
-                write_multi_max_size: Some(5 * 1024 * 1024 * 1024),
+                write_multi_max_size: if cfg!(target_pointer_width = "32") {
```

Comment:
How about check `target_pointer_width = "64"` instead, so we can running on 16 bit platforms?

Reply:
Sure, but rust [officially supported](https://doc.rust-lang.org/nightly/rustc/platform-support.html) 16bit platform is msp430-none-elf, it does come with `no-std` and will break most of our dependencies I'm afraid.